packages/graph/hash_graph/tests/integration/postgres/entity_type.rs

```diff
@@ -1,25 +1,33 @@
 use crate::postgres::DatabaseTestWrapper;
+pub use crate::test_data::{data_type, entity_type, property_type};
 
 #[test]
 fn insert() {
-    let person_et = serde_json::from_str(crate::test_data::entity_type::PERSON_V1)
-        .expect("could not parse entity type");
+    let person_et =
```

Comment:
Just to flag, I really don't like these abbreviations (`et`). That said, I understand why you went with them and I'm not sure I'd go with something else. I think having to write `person_entity_type` isn't great, and I think just going with `person` could get a bit confusing when we start mixing property types, data types, etc.

If anyone has better ideas I'd love to hear them because I really don't like this current style but I can't think of anything better.

Reply:
I had the exact same problem. Especially when mixing property types that simply map to data types it gets confusing (e.g. a Text property type only containing the Text data type). Any suggestions are much appreciated!